src/components/organisms/UserProfileModal/UserProfileModal.scss

```diff
@@ -28,7 +28,7 @@ $profile-link--padding: 6px 18px;
   }
 }
 
-.question {
+.no-margin {
   margin-bottom: 0;
 }
```

Comment:
Can we move this definition inside the `.UserProfileModal` scope rather than being here in global scope?

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/36f9ee2f4cfc1f5342c9ef7c0843ba5452393d9f